packages/transaction-controller/src/utils/validation.ts

```diff
@@ -194,3 +195,129 @@ function validateParamChainId(chainId: number | string | undefined) {
     );
   }
 }
+
+type GasFieldsToValidate = 'gasPrice' | 'maxFeePerGas' | 'maxPriorityFeePerGas';
+/**
+ * Validates gas values.
+ *
+ * @param txParams - The transaction parameters to validate.
+ */
+function validateGasParams(txParams: TransactionParams) {
```

Comment:
Should we call this `validateGasFeeParams` to keep it distinct from the `gas` value itself?

Reply:
Fixed 👍